fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowLoadStmtAnalyzer.java

```diff
@@ -0,0 +1,195 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.analyzer;
+
+import com.google.common.base.Strings;
+import com.starrocks.analysis.BinaryPredicate;
+import com.starrocks.analysis.CompoundPredicate;
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.LikePredicate;
+import com.starrocks.analysis.OrderByElement;
+import com.starrocks.analysis.ShowLoadStmt;
+import com.starrocks.analysis.SlotRef;
+import com.starrocks.analysis.StringLiteral;
+import com.starrocks.cluster.ClusterNamespace;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.common.proc.LoadProcDir;
+import com.starrocks.common.util.OrderByPair;
+import com.starrocks.load.loadv2.JobState;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.AstVisitor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShowLoadStmtAnalyzer {
+
+    public static Void analyze(ShowLoadStmt statement, ConnectContext context) {
+        return new ShowLoadStmtAnalyzerVisitor().visitShowLoadStmt(statement, context);
```

Comment:
In visitor style, should call `visit()` here

Reply:
done